samples/chaincode/cc-tools-demo/README.md

```diff
@@ -0,0 +1,329 @@
+```
+
+Edit the file of the error `$FPC_PATH/samples/chaincode/cc-tools-demo/vendor/github.com/hyperledger-labs/cc-tools/mock/mockstub.go` and add the missing method there:
+
+```go
+ 	// PurgePrivateData ...
+ 	func (stub *MockStub) PurgePrivateData(collection, key string) error {
+ 		return errors.New("Not Implemented")
+ 	}
+
+```
+
+After building again, you can check that the `fpc/fpc-cc-tools-demo` image exists in your local docker registry using:
+
+```bash
+docker images | grep fpc-cc-tools-demo
+```
+
+## Time to test!
+
+Next step is to test the chaincode by invoking transactions, for which you need a basic Fabric network with a channel.
+We will use the test network provided in [`$FPC_PATH/samples/deployment/test-network`](../../deployment/test-network).
+To invoke the chaincode, we will use the `simple-cli` application in [`$FPC_PATH/samples/application/simple-cli-go`](../../pplication/simple-cli-go).
+
+### Prepare the test network
+
+We already provide a detailed tutorial how to use FPC with the test network in [`$FPC_PATH/samples/deployment/test-network`](../../deployment/test-network).
+However, for completeness, let's go through the required steps once again.
+
+```bash
+cd $FPC_PATH/samples/deployment/test-network
+./setup.sh
+```
+
+### Start the test network
+
+Now we are ready to launch the Fabric test network and install the FPC chaincode on it.
+We begin with setting up the network with a single channel `mychannel`.
+
+```bash
+cd $FPC_PATH/samples/deployment/test-network/fabric-samples/test-network
+./network.sh up -ca
+./network.sh createChannel -c mychannel
+```
+
+### Install the chaincode
+
+Once the network is up and running, we install the cc-tools-demo chaincode and the FPC Enclave Registry.
+We provide a small shell script to make this task a bit easier.
+
+```bash
+export CC_ID=cc-tools-demo
+export CC_PATH="$FPC_PATH/samples/chaincode/cc-tools-demo/"
+export CC_VER=$(cat "$FPC_PATH/samples/chaincode/cc-tools-demo/mrenclave")
+cd $FPC_PATH/samples/deployment/test-network
+./installFPC.sh
+```
+
+### Set the needed env vars in the docker-compose file and start the chaincode
+
+From the code above, we need to set two env variables for the chaincode application to work and use FPC and chaincode-as-a-service (CCAAS).
+
+```yaml
+- RUN_CCAAS=true
+- FPC_ENABLED=true
+```
+
+To achieve this, we created extra configurations for the start command at `$FPC_PATH/samples/deployment/test-network/cc-tools-demo-compose.yaml`.
+
+Continue by running:
+
+```bash
+export EXTRA_COMPOSE_FILE="$FPC_PATH/samples/deployment/test-network/cc-tools-demo-compose.yaml"
+make ercc-ecc-start
+```
+
+You should see now four containers running (i.e., `cc-tools-demo.peer0.org1`, `cc-tools-demo.peer0.org2`, `ercc.peer0.org1`, and `ercc.peer0.org2`).
+
+### Invoke simple getSchema transaction
+
+Open a new terminal and connect to the `fpc-development-go-support` container by running
+
+```bash
+docker exec -it fpc-development-go-support /bin/bash
```

Comment:
are you sure about the name of the environment? I had to change fpc-development-go-support to fpc-development-main .... I am guessing you are using a branch and maybe that is why the difference?

Reply:
Done